dynamodb/schema.go

```diff
@@ -128,13 +128,21 @@ func analyzeMetadata(client dynamoClient, s *schema.Table) error {
 		s.PrimaryKeys = append(s.PrimaryKeys, schema.Key{Column: *i.AttributeName})
 	}
 
-	// Secondary indexes
+	// Secondary indexes from GlobalSecondaryIndexes
 	for _, i := range result.Table.GlobalSecondaryIndexes {
 		var keys []schema.Key
 		for _, j := range i.KeySchema {
 			keys = append(keys, schema.Key{Column: *j.AttributeName})
 		}
-		// s.SecIndexes = append(s.SecIndexes, index{Name: *i.IndexName, Keys: keys})
+		s.Indexes = append(s.Indexes, schema.Index{Name: *i.IndexName, Keys: keys})
```

Comment:
Does this mean we treat these two types of indexes in the same way later? Why don't we store GlobalSecondaryIndexes in `SecIndexes`?

Reply:
SecIndexes might be a deprecated field. Currently, the struct field offers only one field for this i.e. `Indexes`.
```
type Table struct {
	Name        string
	ColNames    []string          
	ColDefs     map[string]Column
	PrimaryKeys []Key
	ForeignKeys []ForeignKey
	Indexes     []Index
}
```